matchmaking/backend/config.js

```diff
@@ -0,0 +1,32 @@
+require("dotenv").config({ path: __dirname + "/../../.env" });
```

Comment:
You shouldn't need this as dotenv is required in index.js which makes process.env available to the rest of the app. Also you can't set the dirname to ../../.env as that will only work locally. The its left as default in index.js and grabs the env vars set by the docker-compose and dockerfile

Reply:
Good catch, I'll make the change